website/static/js/profile.js

```diff
@@ -563,13 +582,88 @@ var SocialViewModel = function(urls, modes) {
         return false;
     });
 
+    self.addWebsiteInput = function() {
+        this.profileWebsites.push(ko.observable().extend({
+            trimmedURL: true
+        }));
+    };
+    
+    self.removeWebsite = function(profileWebsite) {
+        var profileWebsites = ko.toJS(self.profileWebsites()),
+            idx = profileWebsites.indexOf(profileWebsite);
+            bootbox.confirm({
+                title: 'Remove website?',
+                message: 'Are you sure you want to remove this website from your profile?',
+                callback: function(confirmed) {
+                    if (confirmed) {
+                        self.profileWebsites.splice(idx, 1);
+                        if (!self.profileWebsites().length) {
```

Comment:
Since you're checking length here, I think `self.profileWebsites().length === 0` would be more clear